nilearn/glm/first_level/first_level.py

```diff
@@ -1,5 +1,4 @@
-"""
-This module contains the GLM and contrast classes that are meant to be the main
+"""Contain the GLM and contrast classes that are meant to be the main \
```

Comment:
```suggestion
"""Contains the GLM and contrast classes that are meant to be the main \
```

Reply:
Will throw a flake8 error for not using the imperative mood.

Reply:
Wait this is a module level docstring: this may actually work.

Reply:
Ok, can also leave as is

Reply:
It actually works: imperative mood is only for methods and functions doc string